Dockerfile

```
@@ -88,6 +88,10 @@ RUN chown -R ${nixuser}:${nixuser} ${HOME}/.openmpi
 # ------------------------------------------------------------
 # Any benchmarks specifics will go here
 # ------------------------------------------------------------
+ADD HPL.dat ${HOME}/
+ADD HPL2.dat ${HOME}/
+ADD run-xhpl.sh ${HOME}/
```

```suggestion
COPY HPL.dat ${HOME}/
COPY HPL2.dat ${HOME}/
COPY run-xhpl.sh ${HOME}/
```

Use `COPY` instead of `ADD` for copying local files into a Docker container